lightning/src/ln/features.rs

```diff
@@ -384,6 +384,9 @@ mod sealed {
 	define_feature!(27, ShutdownAnySegwit, [InitContext, NodeContext],
 		"Feature flags for `opt_shutdown_anysegwit`.", set_shutdown_any_segwit_optional,
 		set_shutdown_any_segwit_required, supports_shutdown_anysegwit, requires_shutdown_anysegwit);
+	define_feature!(31, Taproot, [InitContext, NodeContext, ChannelTypeContext],
```

Comment:
Should we add a `TODO(taproot)` to the module docs above to remember to document support for simple taproot channels when the time comes?